extensions/plugins/system/redcore/redcore.php

```diff
@@ -90,6 +90,11 @@ public function onAfterInitialise()
 						$data = RApi::getPostedData();
 						$dataGet = $input->get->getArray();
 
+						if (!empty($data['language']) && empty($dataGet['language']))
```

Comment:
you should set the input right here because it is just prior the getting the item or list of items, and this is what you have been looking for...